runelite-client/src/main/java/net/runelite/client/plugins/zalcano/ZalcanoConfig.java

```diff
@@ -38,104 +38,116 @@
 {
 
 	@ConfigTitleSection(
-		keyName = "zalcanoTitle",
-		name = "Zalcano",
-		description = "",
-		position = 0
+			keyName = "zalcanoTitle",
+			name = "Zalcano",
+			description = "",
+			position = 0
```

Comment:
I can't quite understand where is this extra indentation is coming from. When i initially submitted code i didn't touch this part of the code at all. 
Gona try copy pasting original and submitting. Nope somehow my intellij adds it?

Reply:
Are you running automatic code refactors or something like that? The checkstyle plugin for example?

Because you probably aren't using the correct checkstyle please have a look here https://github.com/runelite/runelite/wiki/Code-Conventions

Reply:
oh
Well those extra indents, where in the original code allready.